src/main/java/org/terasology/tutorialpathfinding/systems/SpawnSystem.java

```diff
@@ -84,10 +81,10 @@ public void blockchanged(OnChangedBlock event, EntityRef entityRef) {
 
         if (spawner.equals(newBlock)) {
             logger.error("spawner placed");
-            spawnerPositions.add(JomlUtil.from(event.getBlockPosition()));
+            spawnerPositions.add(event.getBlockPosition());
         }
         if(target.equals(newBlock)){
-            targetPostion = new Vector3f(JomlUtil.from(event.getBlockPosition()));
+            targetPostion = new Vector3f(event.getBlockPosition());
```

Comment:
Maybe `targetPosition` (there's a typo btw) should be initialized in the constructor/directly and we just use `.set` here?